internal/formatter/clusters_full.go

```diff
@@ -316,3 +322,19 @@ func (c *clusterInfoRegionsContext) VpcName() string {
 func (c *clusterInfoRegionsContext) MarshalJSON() ([]byte, error) {
 	return json.Marshal(c.clusterInfoRegion)
 }
+
+func AffinitizedRegionToEmoji(isAffinitized bool) string {
+	switch isAffinitized {
+	case true:
+		// Windows terminal do not support emoji
+		// So we return directly the healthstate
+		if runtime.GOOS == "windows" {
+			return fmt.Sprintf("*")
+		}
+		return emoji.Parse(":star:")
+	case false:
+		return fmt.Sprintf("")
+	default:
+		return fmt.Sprintf("")
+	}
+}
```

Comment:
You can simplify in this case

```suggestion
func AffinitizedRegionToEmoji(isAffinitized bool) string {
	if isAffinitized {
		if runtime.GOOS == "windows" {
			return "*"
		}
		return emoji.Parse(":star:")
	}
	return ""
}
```